lib/ws_servers/api/snapshots/send_all.js

```diff
@@ -10,9 +10,7 @@ const sendOrders = require('./send_orders')
 module.exports = async (session, ws) => {
   const { rest } = session.getClient()
 
-  return Promise.all([
-    sendWallets(session, ws, rest),
-    sendPositions(session, ws, rest),
-    sendOrders(session, ws, rest)
-  ])
+  await sendWallets(session, ws, rest)
```

Comment:
is there a specific reason why we're sequentializing this one?

Reply:
in some cases, when switching modes too fast, I've notice some request timeout errors, so the idea here is to reduce the pressure on the api to avoid throttling.